src/content/content.ts

```diff
@@ -18,6 +18,35 @@ const footerComponent = {
   },
 };
 
+const personalisationComponent = {
+  header: 'Personalisation',
+  details: {
+    title: 'Personalisation details',
+    text1:
+      'Use double brackets to add a personalisation field to your content. For example:',
+    codeBlockText: 'Hello ((firstName)), your NHS number is ((NHSnumber))',
+    text2: 'NHS Notify gets data from PDS to populate personalisation fields.',
+    text3: 'You can use:',
+    list: [
+      { id: 'pds-item-1', item: '((fullName))' },
+      { id: 'pds-item-2', item: '((firstName))' },
+      { id: 'pds-item-3', item: '((middleNames))' },
+      { id: 'pds-item-4', item: '((lastName))' },
+      { id: 'pds-item-5', item: '((nhsNumber))' },
+      { id: 'pds-item-6', item: '((birthDate))' },
```

Comment:
not your fault if it's on the ticket design but comms manager does not prepopulate date of birth personalisation - maybe one to pick up with the R+D team?

Reply:
Lets remove it for now and flag this in a comment on the ticket, tagging Liam and Jake.

Reply:
I have discussed this with Alex Colbeck too, he will raise it with their team